src/download.rs

```diff
@@ -111,27 +111,27 @@ fn get_love_download_location(
 ) -> Result<LoveDownloadLocation> {
-    let release_location = "https://bitbucket.org/rude/love/downloads";
-    let release_file_name = match (version, platform, bitness) {
-        (LoveVersion::V11_3, Platform::Windows, Bitness::X64) => "love-11.3-win64.zip",
-        (LoveVersion::V11_3, Platform::Windows, Bitness::X86) => "love-11.3-win32.zip",
-        (LoveVersion::V11_3, Platform::MacOs, Bitness::X64) => "love-11.3-macos.zip",
-
-        (LoveVersion::V11_2, Platform::Windows, Bitness::X64) => "love-11.2-win64.zip",
-        (LoveVersion::V11_2, Platform::Windows, Bitness::X86) => "love-11.2-win32.zip",
-        (LoveVersion::V11_2, Platform::MacOs, Bitness::X64) => "love-11.2-macos.zip",
-
-        (LoveVersion::V11_1, Platform::Windows, Bitness::X64) => "love-11.1-win64.zip",
-        (LoveVersion::V11_1, Platform::Windows, Bitness::X86) => "love-11.1-win32.zip",
-        (LoveVersion::V11_1, Platform::MacOs, Bitness::X64) => "love-11.1-macos.zip",
-
-        (LoveVersion::V11_0, Platform::Windows, Bitness::X64) => "love-11.0.0-win64.zip",
-        (LoveVersion::V11_0, Platform::Windows, Bitness::X86) => "love-11.0.0-win32.zip",
-        (LoveVersion::V11_0, Platform::MacOs, Bitness::X64) => "love-11.0.0-macos.zip",
-
-        (LoveVersion::V0_10_2, Platform::Windows, Bitness::X64) => "love-0.10.2-win64.zip",
-        (LoveVersion::V0_10_2, Platform::Windows, Bitness::X86) => "love-0.10.2-win32.zip",
-        (LoveVersion::V0_10_2, Platform::MacOs, Bitness::X64) => "love-0.10.2-macosx-x64.zip",
+    let release_location = "https://github.com/love2d/love/releases/download";
+    let (version_string, release_file_name) = match (version, platform, bitness) {
+        (LoveVersion::V11_3, Platform::Windows, Bitness::X64) => ("11.3", "love-11.3-win64.zip"),
+        (LoveVersion::V11_3, Platform::Windows, Bitness::X86) => ("11.3", "love-11.3-win32.zip"),
+        (LoveVersion::V11_3, Platform::MacOs, Bitness::X64) => ("11.3", "love-11.3-macos.zip"),
+
+        (LoveVersion::V11_2, Platform::Windows, Bitness::X64) => ("11.2", "love-11.2-win64.zip"),
+        (LoveVersion::V11_2, Platform::Windows, Bitness::X86) => ("11.2", "love-11.2-win32.zip"),
+        (LoveVersion::V11_2, Platform::MacOs, Bitness::X64) => ("11.2", "love-11.2-macos.zip"),
+
+        (LoveVersion::V11_1, Platform::Windows, Bitness::X64) => ("11.1", "love-11.1-win64.zip"),
+        (LoveVersion::V11_1, Platform::Windows, Bitness::X86) => ("11.1", "love-11.1-win32.zip"),
+        (LoveVersion::V11_1, Platform::MacOs, Bitness::X64) => ("11.1", "love-11.1-macos.zip"),
+
+        (LoveVersion::V11_0, Platform::Windows, Bitness::X64) => ("11.0", "love-11.0.0-win64.zip"),
+        (LoveVersion::V11_0, Platform::Windows, Bitness::X86) => ("11.0", "love-11.0.0-win32.zip"),
+        (LoveVersion::V11_0, Platform::MacOs, Bitness::X64) => ("11.0", "love-11.0.0-macos.zip"),
+
+        (LoveVersion::V0_10_2, Platform::Windows, Bitness::X64) => ("10.2", "love-0.10.2-win64.zip"),
```

Comment:
I believe the "10.2" version here is breaking the build, since before `11.x`, the versioning scheme was `0.X.Y`, so it should be `0.10.2`.

This URL gives a 404: https://github.com/love2d/love/releases/download/10.2/love-0.10.2-win32.zip
But this does not: https://github.com/love2d/love/releases/download/0.10.2/love-0.10.2-win32.zip